hooks/common/post-code-deploy/post-code-deploy.sh

```diff
@@ -16,10 +16,9 @@ source_branch="$3"
 deployed_tag="$4"
 repo_url="$5"
 repo_type="$6"
-drush_alias=${site}'.'${target_env}
 
-# Load utility functions.
-. `dirname $0`/../capitalcamp-functions.sh
```

Comment:
Remove capitalcamp-functions.sh from codebase.  They have been replaced by built-in cloudhooks.